gym/envs/box2d/bipedal_walker.py

```diff
@@ -154,12 +155,10 @@ class BipedalWalker(gym.Env, EzPickle):
 
     """
 
-    metadata = {"render_modes": ["human", "rgb_array"], "render_fps": FPS}
+    metadata = {"render_modes": [None, "human", "rgb_array"], "render_fps": FPS}
```

Comment:
This will be just a matter of convention, but do we need to include `None` as a render mode? I think any environment naturally supports a `None` mode, since it basically just says "do whatever, I don't care about rendering".

Reply:
I added it to match `assert render_mode in self.metadata["render_modes"]`, but we can change those lines instead.
Also in tests, if you want to try all the combinations, you can do it with `for mode in env.metadata["render_modes"]`.
Are we sure that all environments want to support None as render mode?

Pointed these out, I don't have a strong preference. I agree that repeating it in all environments is not nice.

Reply:
I removed None in render_modes and changed asserts